packages/phone-number-privacy/signer/src/pnp/endpoints/sign/io.ts

```diff
@@ -66,9 +71,22 @@ export class PnpSignIO extends IO<SignMessageRequest> {
 
   async authenticate(
     request: Request<{}, {}, SignMessageRequest>,
+    warnings: string[],
     logger: Logger
   ): Promise<boolean> {
-    return authenticateUser(request, this.kit, logger, this.shouldFailOpen)
+    const { success, failedOpen } = await authenticateUser(
```

Comment:
same as the note referencing the combiner PNP io code

Reply:
See my comment above